File: lab02/Estudiante.py
from Persona import Persona

class Estudiante(Persona):

    def __init__(self, nombre: str = "", edad: int = 0, carrera: str = ""):
        super().__init__(nombre, edad)
        self._carrera = carrera

    @property
    def carrera(self) -> str:
        return self._carrera

    @carrera.setter
    def carrera(self, valor: str):
        self._carrera = valor

    def __str__(self) -> str:
        return f"Estudiante [{super().__str__()}, Carrera: {self._carrera}]"

File: lab02/Estudiante.java
public class Estudiante extends Persona {
    private String carrera;

    public Estudiante() { super(); }

    public Estudiante(String nombre, int edad, String carrera) {
        super(nombre, edad);
        this.carrera = carrera;
    }
    public String getCarrera() { 
        return carrera; 
    }
    public void setCarrera(String carrera) { 
        this.carrera = carrera; 
    }
    @Override
    public String toString() {
        return "Estudiante [" + super.toString() + ", Carrera: " + carrera + "]";
    }
}